src/platform/mt8186/lib/clk.c

```diff
@@ -37,6 +36,39 @@ STATIC_ASSERT(ARRAY_SIZE(platform_cpu_freq) == NUM_CPU_FREQ,
 
 static SHARED_DATA struct clock_info platform_clocks_info[NUM_CLOCKS];
 
+static void clk_dsppll_enable(uint32_t value)
+{
+	tr_dbg(&clkdrv_tr, "clk_dsppll_enable: %d\n", value);
+
+	switch (value) {
+	case MTK_CLK_ADSP_DSPPLL:
+		io_reg_write(MTK_ADSPPLL_CON1, 0x810F6276);
+		break;
+	case MTK_CLK_ADSP_DSPPLL_2:
+		io_reg_write(MTK_ADSPPLL_CON1, 0x831EC4ED);
```

Comment:
would be better to have these values decoded or at least have an explanatory comment

Reply:
These are magic numbers to divide the PLL clock to desired frequency. I'll replace this part with named macro.